content/en/users/aai/check-in/vos/_index.md

```diff
@@ -68,17 +68,18 @@ A step-by-step guide for the VO registration process is provided in the procedur
 
    ![VO population](./check-in-my-vo-population-result.png)
 
-### Accepting new VO members
+### VO enrollment URLs
+
+Vo enrollments URLs are essential for someone who wants to enroll to a VO.
 
 Users can request membership in your VO by following the VO enrollment URL. The
 enrollment URL has the following form:
 
 `https://aai.egi.eu/registry/co_petitions/start/coef:##` where `##` is the
 unique numeric identifier for the enrollment flow of your VO.
 
-Once a user submits a VO membership petition, all VO managers are notified with
-an email containing a link to the petition. Any of the VO managers can then
-review the petition and either **approve** or **deny** the request.
+The VO Manager can insert this URL in an invitation email and send it directly to the users who want to join the VO.
+Also, it can be used in any registration web page as an href of a button or image etc.
 
 The VO enrollment URL can be found through the EGI Check-in Registry:
 
```

Comment:
Add here that the enrollment URL can be found both on the Operations Portal VO List by quickly searching for the VO name and clicking on the Details and on Check-in Registry ...

Reply:
I couldnt add it exactly like this, because of lint errors, so I splitted it to more sentences. But I think it, its ok.

Reply:
@vardizzo-lab I have made the changes you requested. Is there anything, that is pending for my side?  I think now, remains the question from Baptiste.